arcade.code-workspace

```diff
@@ -1,16 +0,0 @@
-{
-	"folders": [
```

Comment:
why delete this?

Reply:
Oh, I checked that in by accident I'm pretty sure. R u using this?

Reply:
nvm, it was peli. I'll put this back